yb-voyager/cmd/assessMigrationCommandBulk.go

```diff
@@ -0,0 +1,316 @@
+)
+
+var bulkAssessmentDir string
+var fleetConfigPath string
+var continueOnError utils.BoolStr
+var bulkAssessmentReport BulkAssessmentReport
+
+type AssessMigrationDBConfig struct {
+	DbType      string
+	Host        string
+	Port        string
+	ServiceName string
+	SID         string
+	TnsAlias    string
+	User        string
+	Password    string
+	Schema      string
+}
+
+var assessMigrationBulkCmd = &cobra.Command{
+	Use:   "assess-migration-bulk",
+	Short: "Bulk Assessment of multiple schemas across one or more Oracle database instances",
+	Long:  "Bulk Assessment of multiple schemas across one or more Oracle database instances",
+
+	Run: func(cmd *cobra.Command, args []string) {
+		assessMigrationBulk()
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(assessMigrationBulkCmd)
+
+	// defining flags
+	assessMigrationBulkCmd.Flags().StringVar(&fleetConfigPath, "fleet-config-file", "", "File containing the connection params for schema(s) to be assessed (required)")
+	BoolVar(assessMigrationBulkCmd.Flags(), &continueOnError, "continue-on-error", true, "If true, it will print the error message on console and continue to next schema’s assessment")
+	assessMigrationBulkCmd.Flags().Bool("ignore-exists", true, "If true, skip assessment if for a schema it's already done")
+	assessMigrationBulkCmd.Flags().StringVar(&bulkAssessmentDir, "bulk-assessment-dir", "", "Top-level directory storing the export-dir of each schema (default: pwd)")
+	BoolVar(assessMigrationBulkCmd.Flags(), &startClean, "start-clean", false, "Cleans up all the export-dirs in bulk assessment directory to start everything from scratch")
+
+	// marking mandatory flags
+	assessMigrationBulkCmd.MarkFlagRequired("fleet-config-file")
+	assessMigrationBulkCmd.MarkFlagRequired("bulk-assessment-dir")
+}
+
+func assessMigrationBulk() {
+	if startClean {
+		proceed := utils.AskPrompt(
+			"CAUTION: Using --start-clean will delete all progress in each export directory present inside the bulk-assessment-dir. " +
+				"Do you want to proceed")
+		if !proceed {
+			return
+		}
+
+		// cleaning all export-dir present inside bulk-assessment-dir
+		matches, err := filepath.Glob(fmt.Sprintf("%s/*-export-dir", bulkAssessmentDir))
+		if err != nil {
+			utils.ErrExit("error while cleaning up export directories: %s", err)
+		}
+		for _, match := range matches {
+			utils.CleanDir(match)
+		}
+
+		err = os.RemoveAll(filepath.Join(bulkAssessmentDir, "bulkAssessmentReport.html"))
+		if err != nil {
+			utils.ErrExit("failed to remove bulk assessment report: %s", err)
+		}
+	}
+
+	dbConfigs, err := parseFleetConfigFile(fleetConfigPath)
+	if err != nil {
+		utils.ErrExit("failed to parse fleet config file: %v", err)
+	}
+
+	for _, dbConfig := range dbConfigs {
+		utils.PrintAndLog("\nAssessing '%s' schema\n", dbConfig.Schema)
+		err = executeAssessment(dbConfig)
+		if err != nil {
+			log.Errorf("failed to assess migration for schema %s: %v", dbConfig.Schema, err)
```

Comment:
where do these logs go? I'm assuming there is a separate log file for the bulk command store in bulk-dir/logs/